builder/virtualbox/common/run_config.go

```diff
@@ -11,6 +12,13 @@ type RunConfig struct {
 	Headless    bool   `mapstructure:"headless"`
 	RawBootWait string `mapstructure:"boot_wait"`
 
+	HTTPDir     string `mapstructure:"http_directory"`
+	HTTPPortMin uint   `mapstructure:"http_port_min"`
+	HTTPPortMax uint   `mapstructure:"http_port_max"`
```

Comment:
I put these back in while resolving a merge conflict but I think these may have been refactored to go somewhere else. Going to check.